packages/core/src/components/search/SearchInput/SearchInput.tsx

```diff
@@ -106,7 +106,6 @@ const SearchInput = forwardRef<SearchInputRef, SearchInputProps>(
       addToSearchHistory({ term, path })
       sendAnalytics(term)
       setSearchDropdownVisible(false)
-      setSearchQuery(term)
```

Comment:
removing because when clicking in a suggested product, we don't need to update the query.